package.json

```diff
@@ -42,6 +42,8 @@
   },
   "dependencies": {
     "abort-controller": "^3.0.0",
+    "aggregate-error": "^3.0.1",
+    "any-signal": "^1.0.0",
     "async": "^2.6.2",
```

Comment:
🙏 is it time to remove?

Reply:
I've got a cleanup branch that gets rid of all of these old modules, code and turns bundlesize back on: https://github.com/libp2p/js-libp2p/pull/497. I'll add the `it-all` switch to that.